<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDirectoriesForm</class>
 <widget class="QWidget" name="SettingDirectoriesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Git command</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit_git_command"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_select_git_command">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>File command</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="lineEdit_file_command"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_select_file_command">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>GPG command (option)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLineEdit" name="lineEdit_gpg_command"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_select_gpg_command">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Default working folder</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEdit_default_working_dir"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_browse_default_working_dir">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>74</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_git_command</tabstop>
  <tabstop>pushButton_select_git_command</tabstop>
  <tabstop>lineEdit_file_command</tabstop>
  <tabstop>pushButton_select_file_command</tabstop>
  <tabstop>lineEdit_gpg_command</tabstop>
  <tabstop>pushButton_select_gpg_command</tabstop>
  <tabstop>lineEdit_default_working_dir</tabstop>
  <tabstop>pushButton_browse_default_working_dir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
